cassandra/README.md

```diff
@@ -64,6 +64,78 @@ Docker Hub.
 
 <!-- Optional -->
 ### Deploy with Kubernetes
+This image can also be deployed on a Kubernetes cluster.The following example YAML files are provided 
+in the repository as reference for deploying Cassandra with Stateful Sets on a single node cluster:
+
+[`pv-local.yaml`](https://github.com/clearlinux/dockerfiles/blob/master/cassandra/pv-local.yaml): local persistent volumes for cassandra database.
+[`storageclass.yaml`](https://github.com/clearlinux/dockerfiles/blob/master/cassandra/storageclass.yaml): create a StorageClass with the volumeBindingMode set to “WaitForFirstConsumer”
+[`cassandra-statefulset.yaml`](https://github.com/clearlinux/dockerfiles/blob/master/cassandra/cassandra-statefulset.yaml): create a Headless Service and a Statefulset for Cassandra with three replicas.
+
+The example utilies official clearlinux cassandra container, to deploy this container on a Kubernetes cluster,
+please follow with below steps:
+
+1. Create /mnt/disks directory on local test cluster
+    ```
+    $ mkdir /mnt/disks
+    $ for vol in vol1 vol2 vol3; do
+          mkdir /mnt/disks/$vol
+          chmod 777 /mnt/disks/$vol
```

Comment:
The character looks weird.

Reply:
And why did you have jue's cassandra commit in your PR?

Reply:
correct that. please review https://github.com/clearlinux/dockerfiles/pull/260.